ui/src/pages/Login.tsx

```diff
@@ -1,11 +1,11 @@
 import Button from "@material-ui/core/Button";
 import Card from "@material-ui/core/Card";
 import CardContent from "@material-ui/core/CardContent";
+import CircularProgress from "@material-ui/core/CircularProgress";
```

Comment:
Can we make it `import { Button, Card, CardContent, CircularProgress} from "@material-ui/core`

Reply:
I always prefer to explicitly import for component libraries to make sure tree shaking works properly.